### ISSUE TITLE
About the weather removal dataset?

### ISSUE THREAD

**Issue by zhuyr97** [IGITUGraz/WeatherDiffusion#3]
In the [Outdoor-Rain](https://github.com/liruoteng/HeavyRainRemoval)  dataset, There seems to be an overlap between the training dataset and the test dataset in a data?  

Are you split the dataset as default? 

Best,

**Comment by oozdenizci** [comment]
Hi! Thanks for your comment. We are aware of this issue in the Outdoor-Rain dataset and this has been taken care of.

Firstly, the test set is not manually split from the training set linked in the Outdoor-Rain github repo. There is another test set, known with the annotation "Test1", which corresponds to this dataset and we use that. We can also provide a permanent link to this Test1 set, as we are still in the process of putting together the complete repository.

Secondly, as you have mentioned, there also exists an overlap between Test1 and the full training set (image filenames are the same within the range im_0301_X to im_0350_X). Note that though degraded input examples are not identical in the training and Test1 sets, even though these filenames are the same. However the ground truth examples are the same in both sets for these images.

To accommodate for this, we just did not use patches from these training images (in the range im_0301_X to im_0350_X) during optimization. Our weather-specific model on Outdoor-Rain is trained by excluding these instances of the training set manually. Our multi-weather model is trained on the mentioned AllWeather dataset, which already does not contain any of these training images.

I hope this clarifies your question, otherwise let me know!

**Comment by zhuyr97** [comment]
Yes, you really clarify my confusion.
Could you privide the "Test1" soon？ Thanks a lot!
